components/Breadcrumb/src/stories/breadcrumb.bem.stories.mdx

```diff
@@ -0,0 +1,896 @@
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Breadcrumb" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <nav class="denhaag-breadcrumb">
+      <ol class="denhaag-breadcrumb__list" itemscope itemtype="https://schema.org/BreadcrumbList">
+        <li
+          class="denhaag-breadcrumb__item"
+          itemscope
+          itemtype="https://schema.org/ListItem"
+          itemprop="itemListElement"
+        >
+          <a class="denhaag-breadcrumb__link" href="https://example.com/" itemprop="item">
+            <span class="denhaag-breadcrumb__text" itemprop="name">
+              Home
+            </span>
+            <svg
+              width="1em"
+              height="1em"
+              viewBox="0 0 24 24"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+              class="denhaag-icon"
+              focusable="false"
+              aria-hidden="true"
+              shape-rendering="auto"
+            >
+              <path
+                d="M9.293 18.707a1 1 0 010-1.414L14.586 12 9.293 6.707a1 1 0 011.414-1.414l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414 0z"
+                fill="currentColor"
+              ></path>
+            </svg>
+            <meta itemprop="position" content="1" />
+          </a>
+        </li>
+        <li
+          class="denhaag-breadcrumb__item"
+          itemscope
+          itemtype="https://schema.org/ListItem"
+          itemprop="itemListElement"
+        >
+          <a class="denhaag-breadcrumb__link" href="https://example.com/a/" itemprop="item">
+            <span class="denhaag-breadcrumb__text" itemprop="name">
+              Afval
+            </span>
+            <svg
+              width="1em"
+              height="1em"
+              viewBox="0 0 24 24"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+              class="denhaag-icon"
+              focusable="false"
+              aria-hidden="true"
+              shape-rendering="auto"
+            >
+              <path
+                d="M9.293 18.707a1 1 0 010-1.414L14.586 12 9.293 6.707a1 1 0 011.414-1.414l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414 0z"
+                fill="currentColor"
+              ></path>
+            </svg>
+            <meta itemprop="position" content="2" />
+          </a>
+        </li>
+        <li
+          class="denhaag-breadcrumb__item"
+          itemscope
+          itemtype="https://schema.org/ListItem"
+          itemprop="itemListElement"
+        >
+          <a
+            class="denhaag-breadcrumb__link denhaag-breadcrumb__link--current"
+            href="https://example.com/a/b/"
+            aria-current="location"
```

Comment:
According to [WAI ARIA](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html), this should be `aria-current="page"`

Reply:
Applies to the other stories as well

Reply:
Will do